src/runtime/experiment-flags.ts

```diff
@@ -54,4 +54,14 @@ export enum ArticleExperimentFlags {
    * closable popups.
    */
   BACKGROUND_CLICK_BEHAVIOR_EXPERIMENT = 'background_click_behavior_experiment',
+
+  /**
+   * When enabled the rewarded ad wall isn't closable until the ad is viewed.
+   */
+  REWARDED_ADS_CLOSABLE_ENABLED = 'rewarded_ads_closable_enabled',
```

Comment:
The flag name seems quite misleading. It actually disables "closability" when "closable_enabled" is true. Can we rename this something like "always blcoking"?

Reply:
Done. Updated backend CL too.